test/models/comment_test.rb

```diff
@@ -79,4 +79,22 @@ class CommentTest < ActiveSupport::TestCase
       comment.destroy!
     end
   end
+
+  test '.days_paassed_since_the_last_comment_by_submitter?' do
```

Comment:
```suggestion
  test '#days_paassed_since_the_last_comment_by_submitter?' do
```

インスタンスメソッドなのでこっちかもです〜